identity/source/IotIdentityClientV2.cpp

```diff
@@ -406,17 +406,13 @@ namespace Aws
             {
 
                 std::function<void(Aws::Iot::RequestResponse::IncomingPublishEvent &&)> unmodeledHandler =
-                    [options](Aws::Iot::RequestResponse::IncomingPublishEvent &&event)
+                    [options](Aws::Iot::RequestResponse::IncomingPublishEvent &&publishEvent)
                 {
-                    const auto &payload = event.GetPayload();
-                    Aws::Crt::String objectStr(reinterpret_cast<char *>(payload.ptr), payload.len);
-                    Aws::Crt::JsonObject jsonObject(objectStr);
-                    if (!jsonObject.WasParseSuccessful())
+                    T modeledEvent;
+                    if (!s_initModeledEvent(publishEvent, modeledEvent))
```

Comment:
This doesn't appear to have a paired `s_initModeledEvent` in the `IotIdentityClientV2` file. Is this a generation error?